SCF/OOP_Attempt/Epack.py

```diff
@@ -0,0 +1,284 @@
+#SCF Class definitions
+import time
+import numpy as np
+import psi4
+import scipy.linalg as sl
+from JK_Builder import JK_Builder
+
+class Epack:
+    
+    def __init__(self, wfn, dE_tol, dRMS_tol, maxmacro, SOSCF_tol, maxmicro):#param for SCF iterations, energy types specified below.
```

Comment:
You can set defaults like `dE_tol=1.e-7` right in the input. Especially good for something like SOSCF incase its not used.